react-components/src/tools/requests/NewToolRequestForm.js

```diff
@@ -0,0 +1,145 @@
+                                       component={FormMultilineTextField}/>
+                                <ErrorMessage name="description" component="div"/>
+                                <Field name="source_url"
+                                       label={getMessage("toolSrcLinkLabel")}
+                                       required={true}
+                                       margin="dense"
+                                       validate={this.validateUrl}
+                                       component={FormTextField}/>
+                                <ErrorMessage name="source_url" component="div"/>
+                                <Field name="version"
+                                       label={getMessage("toolVersionLabel")}
+                                       required={true}
+                                       margin="dense"
+                                       component={FormTextField}/>
+                                <ErrorMessage name="version" component="div"/>
+                                <Field name="documentation_url"
+                                       label={getMessage("toolDocumentationLabel")}
+                                       required={true}
+                                       margin="dense"
+                                       validate={this.validateUrl}
+                                       component={FormTextField}/>
+                                <ErrorMessage name="documentation_url" component="div"/>
+                                <Field name="cmd_line"
+                                       label={getMessage("toolInstructionsLabel")}
+                                       required={false}
+                                       margin="dense"
+                                       component={FormMultilineTextField}/>
+                                <ErrorMessage name="cmd_line" component="div"/>
+                                <Field name="test_data_path"
+                                       label={getMessage("toolTestDataLabel")}
+                                       required={false}
+                                       margin="dense"
+                                       component={FormTextField}/>
+                                <ErrorMessage name="test_data_path" component="div"/>
+
+
+                                <Button style={{float: "right"}}
+                                        variant="contained"
+                                        color="primary"
+                                        type="submit"
+                                        disabled={isSubmitting}>
+                                    {getMessage("submit")}
+                                </Button>
+                            </Form>
+                        )}
+                    />
+                </DialogContent>
+            </Dialog>
+        );
+    }
+}
+
+export default withStyles(exStyles)(withI18N(injectIntl(NewToolRequestForm), intlData));
```

Comment:
I don't think any styles are actually used in this file and the styles file under /tools is empty as well.  Maybe we can scrub these out?